scripts/update-windows.ps1

```
@@ -0,0 +1,56 @@
+    FfmpegPackageId = $FfmpegPackageId
+}
+
+if ($ForceUpdate) {
+    $installArguments.ForceUpdate = $true
+}
+
+if ($SkipFfmpegInstall) {
+    $installArguments.SkipFfmpegInstall = $true
+}
+
+if ((Test-Path -LiteralPath $venvPython) -and (& $venvPython -m pip show discord.py *> $null; $LASTEXITCODE -eq 0)) {
```

PSScriptAnalyzer (1.25.0)

[error] Missing closing ')' in expression.

(MissingEndParenthesisInExpression)